src/traces/contour/plot.js

```diff
@@ -244,16 +244,19 @@ function makeLinesAndLabels(plotgroup, pathinfo, gd, cd0, contours, perimeter) {
             .attr('data-notex', 1)
             .call(Drawing.font, contours.labelfont);
 
+        var _x = cd0.x.slice().sort(Lib.sorterAsc);
+        var _y = cd0.y.slice().sort(Lib.sorterAsc);
```

Comment:
non-⛔️ but if we only need the bounds we can do better than a full sort of the arrays.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/3934/commits/9f6d1b63a0f9e2bc7676847d099937c8a868033e.